src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
+public interface backgroundTaskManagerInterface { 
+/**
+   * Checks VM status to make sure they are healthy. 
```

Comment:
what happens if they aren't healthy? if nothing, then I think the comment is fine as is

Reply:
Probably just going to call updateInstance instead, Chuck mentioned it in a comment below, if you are interested

Reply:
that makes sense, thank you!